controllers/Controller.js

```diff
@@ -0,0 +1,132 @@
+const Model = require('../models')
+const View = require('../views/View')
+
+class Controller {
+    static setTable (table) {
+        this.table = table[0].toUpperCase() + table.slice(1)
+        this.fields = Object.keys(Model[this.table].rawAttributes)
+    }
+
+    static setParams (params) {
+        this.params = params
+    }
+
+    static help () {
+        View.help()
+    }
+
+    static add () {
+        let instance = this.fields
+        .filter(field => !['id', 'createdAt', 'updatedAt'].includes(field))
+        .reduce((acc, field, i) => {
+            acc[field] = this.params[i]
+            return acc
+        }, {})
+
+        Model[this.table]
+        .create(instance)
+        .then(instance => {
+            View.logInstance(instance.get({ plain: true }))    
+        })
+        .catch(err => {
+            View.logErr(err)
+        })
+    }
+
+    static readOne () {
+        let id = this.params[0]
+        
+        Model[this.table]
+        .findByPk(id)
+        .then(instance => {
+            if (instance == null) {
+                View.instanceNotFound(id, this.table)
+            } else {
+                View.logInstance(instance.get({ plain: true }))
+            }
+        })
+        .catch(err => {
+            View.logError(err)
+        })
+    }
+
+    static readAll () {
+        Model[this.table]
+        .findAll()
+        .then(instances => {
+            View.logInstances(instances.map(i => i.get({ plain: true })))
+        })
+        .catch(err => {
+            View.logError(err)
+        })
+    }
+
+    static update () {
+        let id = this.params[0]
+        let field = this.params[1]
+        let value = this.params[2]
+
+        Model[this.table]
+        .findByPk(id)
+        .then(instance => {
+            instance[field] = value
+            return instance.save()
+        })
+        .then(_ => {})
+        .catch(err => {
+            View.logError(err)
+        })
+
+    }
+
+    static delete () {
+        let id = this.params[0]
+
+        Model[this.table]
+        .findByPk(id)
+        .then(instance => instance.destroy())
+        .then(_ => {})
+        .catch(err => {
+            View.logError(err)
+        })
+    }
+
+    static findFreshArticle () {
+        if (this.table === 'Article') {
+            Model.Article
+            .findAll()
+            .then(articles => {
+                View.logInstances(articles.filter(article => {
+                    let pastMonth = new Date()
+                    pastMonth.setMonth(pastMonth.getMonth() - 1)
+
+                    return new Date(article.createdAt) >= now
+                }).map(article => article.get({ plain: true })))
+            })
+            .catch(err => {
+                View.logError(err)
+            })
+        }
+    }
+
+    static findUnderAgeAuthor () {
+        if (this.table === 'Author') {
+            Model.Author
+            .findAll({ where: { [Model.Sequalize.Op.lt]: 17 }})
```

Comment:
```suggestion
            .findAll({ where: { age: { [Model.Sequalize.Op.lt]: 17 }}})
```
apakah tanpa diinput age bisa jalan?